octodns_googlecloud/__init__.py

```diff
@@ -17,6 +17,21 @@
 __version__ = __VERSION__ = '0.0.3'
 
 
+def add_trailing_dot(value):
+    """
+    Required function to handle cases where records being pushed
+    to Google Cloud DNS do not end with a dot.
+
+    :param value: Contains the record value
+    :type  value: str
+
+    :type return: str
+    """
+    if value[-1] != '.':
+        value = value + '.'
```

Comment:
Minor nit, things aim to use f-strings now that we're py3+.

```suggestion
        value = f'{value}.'
```